data/GTA3.WidescreenFix/scripts/GTA3.WidescreenFix.ini

```diff
@@ -3,15 +3,15 @@ ResX = 0 // This setting can override game's horizontal resolution (-1|0|*).
 ResY = 0 // This setting can override game's vertical resolution (-1|0|*).
 ForceAspectRatio = auto // This setting can override game's aspect ratio(auto|4:3|5:4|16:9|*:*).
 FrontendAspectRatio = auto // This setting is used to properly scale frontend textures, such as menu background or loading screens(auto|16:9|1:1|*:*).
-HudWidthScale = 0.0 // 0.8 by default.
-HudHeightScale = 0.0 // 0.8 by default.
-RadarWidthScale = 0.0 // 0.9 by default.
-SubtitlesScale = 0.0 // 0.8 by default. This can increase or decrease subtitle's size.
+HudWidthScale = 0.0 // 0.8 by default. For original scaling, set to 1.0
+HudHeightScale = 0.0 // 0.8 by default. For original scaling, set to 1.0714285
+RadarWidthScale = 0.0 // 0.9 by default. For original scaling, set to 1.0
+SubtitlesScale = 0.0 // 0.8 by default. This can increase or decrease subtitle's size. For original scaling, set to 1.0
 FixVehicleLights = 1 // Vehicle lights is too big and you might wanna fix that(1|0).
 RestoreCutsceneFOV = 1 // Original FOV value will be used in cutscenes(1|0).
 DontTouchFOV = 0 // This setting can disable any FOV related fixes, but it's not recommended(1|0).
 HideAABug = 1 // Drawing 1px line at top and left sides of the screen to hide antialiasing bug(1|0|2). (2) creates 4 lines to make picture symmetrical.
 SmartCutsceneBorders = 1 // Same as in VCS PCE, cutscene borders' size will be adjusted to fit your resolution. On resolutions bigger than 16:9 vertical borders will be used. Note that you can enable or disable cutscene borders via 'WIDESCREEN'/'BORDERS' option in game menu(1|0).
 IVRadarScaling = 1 // Makes radar size the same as in GTA IV(1|0).
 ReplaceTextShadowWithOutline = 2 // Replaces text shadow with outline(0|1|2).
-ForceMultisamplingLevel = 0 // Enables antialiasing. Requires asi loader(max|0|*). Not available for steam version.
\ No newline at end of file
+ForceMultisamplingLevel = 0 // Enables antialiasing. Requires asi loader(max|0|*). Not available for steam version.
```

Comment:
GTA3 doesn't have this option, why add an ini setting?

Reply:
It has always been there? I'm not adding it.

Reply:
Oh, for some reason I thought it wasn't in III. Nevermind then.